source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,131 @@ Hardware Checklist
+
+   * - :octicon:`circle`
+     - Four nodes or servers
+
+   * - :octicon:`circle`
+     - | SATA/SAS drives for capacity and NVMe SSDs for high-performance
+       | MinIO recommends a minimum of 8 drives per server
+
+   * - :octicon:`circle`
+     - | 25GbE network for capacity 
+       | 100GbE Network interface cards for high performance
+
+.. important:: 
+
+   Object storage operations are primarily throughput bound. 
+   MinIO takes full advantage of the modern hardware improvements such as AVX-512 SIMD acceleration, 100GbE networking, and NVMe SSDs when available.
+
+Hardware Tests to Complete
+--------------------------
+
+MinIO Diagnostics
+~~~~~~~~~~~~~~~~~
+
+Run the following command.
+If you have access to :ref:`SUBNET <minio-docs-subnet>`, you can upload the results there.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support diag ALIAS
+
+Drive Tests
+~~~~~~~~~~~
+
+Run these tests outside of MinIO to verify capabilities of your hardware.
+Test each drive independently on all servers to ensure they are identical in performance.
+Record the results for later reference.
+
+#. Uncached write test
+   
+    .. code-block::
+       :class: copyable
+
+
+       dd if=/dev/zero of=/mnt/driveN/testfile bs=128k count=80000 oflag=direct conv=fdatasync > dd-write-drive1.txt
+
+#. Uncached read test
+
+    .. code-block::
+       :class: copyable
+
+       dd if=/mnt/diskN/testfile of=/dev/null bs=128k iflag=direct > dd-read-drive1.txt
+
+   Replace ``diskN`` with the path for the disk you are testing.
+
+   If you need to create a sufficiently sized file, you can use the ``head`` command.
+   The following command example creates a 10 Gigabyte file called ``testfile``.
+
+    .. code-block:: shell
+       :class: copyable
+
+       head -c 10G </dev/urandom > testfile
+
+#. IO Controller test
+   
+   Use `IOzone <http://iozone.org/>`__ to test the input/output controllwer and all drives in combination.
+   Document the performance numbers for each server in your deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      iozone -s 1g -r 4m -i 0 -i 1 -i 2 -I -t 160 -F /mnt/sdb1/tmpfile.{1..16} /mnt/sdc1/tmpfile.{1..16} /mnt/sdd1/tmpfile.{1..16} /mnt/sde1/tmpfile.{1..16} /mnt/sdf1/tmpfile.{1..16} /mnt/sdg1/tmpfile.{1..16} /mnt/sdh1/tmpfile.{1..16} /mnt/sdi1/tmpfile.{1..16} /mnt/sdj1/tmpfile.{1..16} /mnt/sdk1/tmpfile.{1..16} > iozone.txt
+
+   .. list-table::
+      :widths: auto
+      :width: 100%
+
+      * - ``-s 1g``
+        - Size of 1G per file
+      * - ``-r`` 
+        - 4m  4MB block size
+      * - ``-i #``   
+        - 0=write/rewrite, 1=read/re-read, 2=random-read/write
+      * - ``-I``     
+        - Direct-IO modern
+      * - ``-t N``   
+        - Number of threads (:math:`numberOfDisks * 16`)
+      * - ``-F <>``  
+        - list of files (the above command tests with 16 files per disk)  
+
+MinIO Tests
+~~~~~~~~~~~
+
+Run the three :mc:`mc support perf` tests.
```

Comment:
Provide examples below, even if we have more examples on the linked page. Esp. in case we add/remove tests later. Easier to say "Perform each of the following :mc:`mc support perf` tests.

Reply:
Copied the examples from the mc support perf docs over.